File: clone.py
from os import write
import subprocess
import string
for i in range(34400) :
    sbd = 29000001 + i
    cmd = "https://dantri.com.vn/thpt/3/0/99/"+ str(sbd) +"/2021/0.25/search-gradle.htm"
    res = subprocess.check_output(["curl", cmd])
    res = str(res)
    filetho = open('a.txt', 'a')
    filetho.write(res+'\n')
    filetho.close()

File: clone.cpp
#include <iostream>
#include <fstream>
#include <string>
#include "cmd.h"
int main() {

    std::fstream res("a.txt", std::ios::app);
    for (int i = 29034400; i < 29034481; i++) {
        std::string sbd = std::to_string(i);
        std::string url_ = "https://dantri.com.vn/thpt/3/0/99/" + sbd + "/2021/0.25/search-gradle.htm";
        std::string text = Command::exec("curl " + url_).output;
        res << "\n" << text;
    }
    res.close();
}